src/format/opamFile.ml

```diff
@@ -1202,16 +1264,39 @@ module ConfigSyntax = struct
       (fun (fld, ppacc) -> fld, Pp.embed with_wrappers wrappers ppacc)
       Wrappers.fields
 
-  let pp =
+  let pp_cond ?f ?condition () =
     let name = internal in
     Pp.I.map_file @@
+    Pp.I.check_opam_version () -|
```

Comment:
The 2.1 version pass `~f` here - the optional argument above wants removing for the 2.0 version (it's triggering a warning)